packages/core/database/src/global.d.ts

```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line no-var
-declare var strapi: any;
+declare var strapi: any & { dirs: { app: { root } } };
```

Comment:
This should be defined in the strapi typings

Reply:
May I ask for help?  

The `@strapi/database` package dose not depend on `@strapi/types`, so I cannot `import { Strapi } from '@strapi/types/src';`. 
Even I can add `@strapi/types` to `@strapi/database`'s devDependencies, it causes a circular dependencies.
So what should I do ?

Reply:
Or maybe someone could update the `strapi` defined in `global.d.ts` first so I could follow it.

Reply:
Sorry, I didn't pay enough attention before to realize what was happening. If this is already defined as 'any' why do we need to add anything to it? Shouldn't it work without changing anything here?

Reply:
It works without this change, I added it for better DX when writing unit tests..
Then should we keep it or remove it ?